megamek/src/megamek/client/ui/swing/util/UIUtil.java

```diff
@@ -897,7 +900,14 @@ private static String sizeString(float deltaScale) {
     public static String colorString(Color col) {
         return " COLOR=" + Integer.toHexString(col.getRGB() & 0xFFFFFF) + " ";
     }
-    
+
+    public static Icon loadIcon(MegaMekFile file, int size) {

```

Comment:
> There's an easy to use File setup that allows users to have their own easily copyable data set from version to version in a subfolder? If so, I'd love to learn about it, cause I've never heard of one.

It's pretty trivial, see 0ccada6 (from #3148) ;)

> Otherwise, looks good.

Ta!